### DISCUSSION TITLE
how to retrieve object push by panorama on local firewall ?

### DISCUSSION THREAD

**Discussion by kevinhuy** [PaloAltoNetworks/pan-os-python#391] [General]
Hello @shinmog  @btorresgil 

i would like to retrieve on local firewall all objects (service , address, address group ) that is push from panorama . i didn't find any option for that. is there a trick for that ?

thanks for your answer :)

**Comment by shinmog** [comment]
The XPATH for config that is on the firewall because of Panorama is different from what's locally configured.  Getting the Panorama pushed config is currently not supported.  This means that the best way to get the answer is to get the answer from Panorama itself.

**Reply by kevinhuy** [reply]
Thanks for your anwser
